### PR TITLE
Changes for Selenium 4 support

### PR DESCRIPTION
**Pull request by nagpalkaran95**

### CODE REVIEW

File: src/test/local/codecept.conf.js [hunk 1]
```diff
@@ -13,27 +13,27 @@ exports.config = {
             user: BROWSERSTACK_USERNAME,
             key: BROWSERSTACK_ACCESS_KEY,
             browser: 'chrome',
```
Comment (07souravkunda) [inline src/test/local/codecept.conf.js:15]: browserName?

File: src/test/parallel/codecept.conf.js [hunk 2]
```diff
@@ -32,26 +46,26 @@ exports.config = {
...
-                     },
-                     "browserVersion": "latest",
-                    },
                     */
                 },
 
                 {
                     browser: "Firefox",
+                    //For W3C-based scripts, use the following capabilties:
+                    desiredCapabilities: {
```
Comment (07souravkunda) [inline src/test/parallel/codecept.conf.js:55]: capabilities?

**Review by 07souravkunda — CHANGES_REQUESTED** [review]

File: src/test/local/codecept.conf.js (hunk 1 above)
Comment (nagpalkaran95) [reply src/test/local/codecept.conf.js:15]: not here. codecept is not accepting browserName. this is codecept syntax

Comment (nagpalkaran95) [reply src/test/local/codecept.conf.js:15]: Capabilities object is w3c

File: src/test/parallel/codecept.conf.js (hunk 2 above)
Comment (nagpalkaran95) [reply src/test/parallel/codecept.conf.js:55]: done. although it was working with that as well

**Review by 07souravkunda — APPROVED** [review]

**Review by francisf — APPROVED** [review]

**Pull request merged**